libs/gretty-runner-jetty/src/main/groovy/org/akhikhl/gretty/JettyServerConfigurer.groovy

```diff
@@ -62,22 +62,11 @@ class JettyServerConfigurer {
 
     def server = configurer.createServer()
 
-    File baseDir = new File(params.baseDir)
-    new File(baseDir, 'webapps').mkdirs()
+    new File(new File(params.baseDir), 'webapps').mkdirs()
```

Comment:
I assume there is no difference to `new File(params.baseDir, 'webapps').mkdirs()`, which is even shorter ;)

Anyhow, thanks for the cool contribution and nice PR / testcase!

Reply:
@aindlq - could you please revert the change on this line and then we can merge the PR?

Reply:
done